packages/shared/utils/countdownCircleTimerDefaultProps.js

```diff
@@ -1,6 +1,7 @@
 export const countdownCircleTimerDefaultProps = {
   size: 180,
   strokeWidth: 12,
+  trailStrokeWidth: false,
```

Comment:
I'd suggest not adding a default value here because that will break peoples snapshot tests. Lets just keep it undefined if it is not provided.